src/display.c

```diff
@@ -96,7 +96,6 @@ void display_init( resolution_t res, bitdepth_t bit, uint32_t num_buffers, gamma
     /* Can't have the video interrupt happening here */
     disable_interrupts();
 
-    /* Minimum is two buffers. */
```

Comment:
I think updating the comment is preferable to removing it. While code obviously takes precedence over comments in terms of information value, a comment as a snapshot in time serves as a possible reminder of any assumptions, motivations, or limitations that were held at the time the comment was written. Someone looking at this code 5 years from now might struggle to imagine why the code is doing what it's doing if there is no comment attached.